src/app/globals.css

```diff
@@ -1,3 +1,20 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+p {
+    margin-bottom: 1em;
+    text-wrap: pretty;
+
+    a {
+        text-decoration: underline;
+        text-decoration-style: dotted;
```

Comment:
I usually see dotted used for tooltips rather than links, so I think we should use the standard underline

Reply:
+1 on this, I also believed the dotted underlines were for tooltips for a split second before I hovered over the link